util/lib.py

```
@@ -342,41 +343,69 @@
+
+    # Output
+    callstring.extend(["merge", "--output", o_out_path])
     logging.info(callstring)
-    
+
     try:
         check_call(callstring)
-    except Exception as e:
-        raise Exception(f"Preparing {o_path} for dashboard failed...", e)
+    except subprocess.CalledProcessError as e:
+        logging.exception("Preparing %s for dashboard failed: %s", o_path, str(e))
```

Ruff

Redundant exception object included in `logging.exception` call

(TRY401)